week-1-practise/level-3/1-piping.js

```diff
@@ -11,10 +11,12 @@
 */
 
 var startingValue = 5;
+var result = function() {
+  return "£" + (startingValue * 2 + 10).toFixed(2);
```

Comment:
I think the question would like to test that you can make use of the `add`, `multiply`, and `format` functions on lines 20, 24 and 28 to produce the correct result.

Can you push a new solution that does this?

Also: you don't need to wrap it all in a function. This is perfectly valid:
```js
var result = "£" + (startingValue * 2 + 10).toFixed(2);
```